packages/cli/test/dev/utils.ts

```diff
@@ -347,7 +347,9 @@ export async function testFixture(
   };
 
   return {
-    dev,
+    dev: dev as any as Omit<typeof dev, 'kill'> & {
+      kill: () => Promise<{ stdout: string; stderr: string }>;
+    },
```

Comment:
Also tried:
```ts
return { 
	dev: {
	  ...dev,
	  kill: () => { 
	    // implementation here 
	  }
	}
```
But, copying dev like this breaks behavior.

Don't love this solution, but `integration-1.test.ts` needs to know the new correct type of `dev.kill`.

One other idea: `testFixture` could return `kill`, `stdout`, and `stderr` as fields directly, and `integration-1.test.ts` could use those instead.

Reply:
This is clunky, but seems to be accurate. I'm good with this for this PR. We can revisit elsewhere.